chesse20.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Arrays;
+import java.util.List;
+import Envy_drocco30.Users.GiveVbucks || Green_Hank.Users.GiveVbucks;
+import goldmine4realxx.Users.GiveVbucks;
+import GETDELETED_WNDR.Users.GiveVbucks;
```

Comment:
@Waynegamer777 Updated the name in the script

Reply:
Can you please explain how all this works and how I will get vbucks? What am I supposed to do?  Thanks

Reply:
When are they adding the pr and what is a pr😂. Sorry if I'm disturbing

Reply:
Last question I promise

Reply:
Pls can you add me to that scrtip too?
My epic is LFS_MatriZ
And how it actually works?
And also how much vbucks it gives me

Reply:
I don't even know how this works and how many vbucks we are getting. I just got my name added.  Ask cheese20 to add you

Reply:
it adds vbucks up to number.huge so its a lot we have to wait for epic games to pull the pr